python_modules/dagster/dagster/components/cli/list.py

```diff
@@ -90,9 +96,20 @@ def list_all_components_schema_command(entry_points: bool, extra_modules: tuple[
     "--output-file",
     help="Write to file instead of stdout. If not specified, will write to stdout.",
 )
+@click.option(
+    "--verbose",
+    "-v",
+    flag_value=True,
+    default=False,
+    help="Show verbose stack traces, including system frames in stack traces.",
+)
```

Comment:
@smackesey does the existance of this flag interact weirdly with any of the logging changes you are planning to make?

Reply:
I can also just drop this flag, it's never invoked right now

Reply:
>does the existance of this flag interact weirdly with any of the logging changes you are planning to make?

this is in `dagster-components`, so no (my logging changes are in dg-land)-- fine to leave it in